src/components/ConsumableExerciseCard.tsx

```diff
@@ -13,95 +15,142 @@ import {
+                return(<SetBlock 
+                    currentWorkoutKey={currentExercise.consumableWorkoutId}
+                    workoutKey={exercise.consumableWorkoutId}
+                    key={index} 
+                    index={index}
+                    setNumber={'warmup' in ref && ref.warmup ? ++numWarmups : ++numWorking} 
+                    setRef={ref}
+                    reps={data?.reps}
+                    weight={data?.weight}
+                    bodyweight={data?.bodyweight}
+                    onChange={updateExerciseCardItem}
+            />)}
+            else if('minutes' in ref && 'seconds' in ref) {
+                return(<RestBlock key={index} minutes={ref.minutes} seconds={ref.seconds}/>);
+            }
+            else {
+                return(<NoteBlock key={index} note={'text' in ref ? ref.text : ''}/>);
+            }
+        }) || [])
+    });
+    useEffect(() => {
+        renderConsumableExerciseItems(exercise);
+    }, [exercise, rerender]);
 
-  // Load font
-  const [fontsLoaded] = useFonts({
-    BebasNeue_400Regular,
-  });
 
-  // Load font
-  useFonts({
-    BebasNeue_400Regular,
-  });
+    // Load font
+    useFonts({
+        BebasNeue_400Regular,
+      });
+    
+      // Load font
+      useFonts({
+        BebasNeue_400Regular,
+      });
+    
+    const updateExerciseCardItem = (index, payload: Partial<ConsumableExerciseData>) => {
 
-  const updateExerciseCardItem = (index, payload: Partial<ConsumableExerciseData>) => {
-    setExercise((oldExercise) => {
-      if (oldExercise.exerciseItems) {
-        const oldData = oldExercise.exerciseItems[index].data;
-        oldExercise.exerciseItems[index].data = {
-          reps: oldData?.reps || '0',
-          weight: oldData?.weight || '0',
-          bodyweight: oldData?.bodyweight || false,
-          ...payload,
-        };
-      }
-      updateConsumableExercise({ consumableWorkoutId, exerciseName, payload: oldExercise });
-      return oldExercise;
-    });
-    setRerender((_rerender) => !_rerender);
-  };
+        setExercise((oldExercise) => {
+          if (oldExercise.exerciseItems) {
+            const oldData = oldExercise.exerciseItems[index].data;
+            oldExercise.exerciseItems[index].data = {
+              reps: oldData?.reps || '0',
+              weight: oldData?.weight || '0',
+              bodyweight: oldData?.bodyweight || false,
+              ...payload,
+            };
+          }
+          updateConsumableExercise({ consumableWorkoutId, exerciseName, payload: oldExercise });
+          return oldExercise;
+        });
+        
+      };
+
+    
+    return (
+        <><ScrollView className='flex-1 w-full h-full p-4 bg-white '>
+                <View className='flex-row justify-between w-full h-10 mb-2 border-b content-evenly border-slate-200'>
+                    <Text className="m-1 mt-3 ml-3 font-bold text-center ">{exercise.exerciseName}</Text>

```

Comment:
I feel like we need another indicator visually to show that someone is viewing a past exercise and not the current one. Can you add a chip or tag above or below the exercise name that says like 'OLD' when we're viewing past exercises? Maybe design something out on figma first. it doesn't have to be much, just another visual indicator (change of color or text somewhere)